Jenkinsfile

```diff
@@ -0,0 +1,93 @@
+pipeline {
+
+  agent any
+
+  triggers {
+    pollSCM('* * * * *')
+    cron('@daily')
+  }
+
+  stages {
+    stage('Install') {
+      agent {
+        docker {
+          image 'node:4.2.1'
+          args "-v /var/lib/jenkins/.npm:/tmp/.npm -e HOME=/tmp"
+          reuseNode true
+        }
+      }
+      steps {
+        sh 'git config --global core.logallrefupdates false'
+        sh 'rm -fr node_modules'
+        checkout([$class: 'GitSCM', branches: [[name: '*/master']], extensions: [[$class: 'RelativeTargetDirectory', relativeTargetDir: '_docker-runner'], [$class: 'CloneOption', shallow: true]], userRemoteConfigs: [[credentialsId: 'GIT_DEPLOY_KEY', url: 'git@github.com:sharelatex/docker-runner-sharelatex']]])
```

Comment:
Why into `_docker-runner` rather than `node_modules/docker-runner-sharelatex`?

Reply:
because I'm using npm to install it from a folder - I've I just cloned it into node_modules directly, it wouldn't install the dependencies etc..